packages/graphql-mocks/src/resolver-map/embed.ts

```diff
@@ -77,15 +78,21 @@ export function embed({
         );
       }
 
-      const wrappedResolver = await applyWrappers(resolverToEmbed, [...wrappers, embedPackOptionsWrapper], {
-        type,
-        field,
-        schema,
-        resolverMap,
-        packOptions,
-      });
+      if (wrappers?.length) {
```

Comment:
Only when there are wrappers should `applyWrappers` be called, otherwise it's an unnecessary step.